float64 velocity
float64 low_level_gain
bool switching
std_msgs/String[] controllers
float64[] gains
